### ISSUE TITLE
bug: Audio focus won't release after notification, leaving media volume silenced

### ISSUE THREAD

**Issue by wwwjj2** [crdroidandroid/issue_tracker#192]
### Issue type

Feature not working as intended

### Device

POCO X3 NFC (surya)

### crDroid version

crDroid 9

### Exact version / Build date

crDroid 9.5 (Android 13)
Build of 2023-05-19

### Bug description

When playing any media content (Music, YouTube video, Podcast, etc.) and notification arrives, the audio focus taken by that notification won't ever release and Music/YouTube sound becomes silenced forever (with music Pause+Play fixes that, with YouTube nothing does and it stays silenced forever)

### Steps to reproduce

Preconditions:
- No Sound-Off Mode, No Do Not Disturb Mode, Sound is ON, Media volume > 0

Steps:
1. Play any Media like Music, YouTube, etc.
2. Receive a notification

What should happen: Audio focus taken by that notification silences the media volume for a while (1-2 seconds) and releases focus after that time making Media volume as it was before the notification

What happens: Audio focus never releases leaving Media volume silenced

### Relevant log (logcat/build log)

```shell
The Logcat is too long for Guthub ("Comment is too long", "There was an error creating your issue: body is too long, body is too long (maximum is 65536 characters)")

Here it is: https://pastebin.com/sdTKFKv7
```

### Additional context

The Logcat Log has the following actions:
1. Press Play on a video in a YouTube Revanced app
2. Receive a Telegram notification
3. Here we go, bug reproduced

### Acknowledgements

- [X] I've checked device is officially supported (for device specific reports and not source related).
- [X] I'm running latest version available on crdroid.net for this device
- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by wwwjj2** [comment]
Note: System screen recording software records sound with no changes - media is not silenced on the record though it is actually silenced through the speakers / bluetooth earphones

**Comment by wwwjj2** [comment]
Update: Not fixed in 20230605 build. It happens rarely but it does. I'll try to get logcat if I can...
